lib/simulator-xcode-6.js

```diff
@@ -901,7 +902,7 @@ class SimulatorXcode6 extends EventEmitter {
    *
    * @return {boolean} True if Touch ID Enrollment menu item is checked in Simulator menu
    */
-  async isTouchIDEnrolled () {
+  async isBiometricEnrolled () {
```

Comment:
I assume merging this PR will require to raise the major version of this lib

Reply:
It will, yes.